model.rb

```diff
@@ -8,18 +8,21 @@ def initialize
   end
 
   # Creates database file if it doesn't exist, sets results to be outputted as
-  # hashes, turns on foreign key support, and creates tables if they don't exist
+  # hashes, turns on foreign key support, creates tables if they don't exist,
+  # and adds initial manager if none exist
   def initialize_database
     @db = SQLite3::Database.new("bank.db")
     db.results_as_hash = true
     db.execute("PRAGMA foreign_keys = ON")
     accounts_table_exists = db.execute("SELECT 1 FROM sqlite_master WHERE 
       type='table' AND name= ?", "accounts").length > 0
+    managers_exist = db.execute("SELECT * FROM managers").length > 0
```

Comment:
Kept getting a an sqlite error from this line. Was able to fix onhttp://stackoverflow.com/questions/32194515/sqlite-error-no-such-table mine by entering the following code onto line 19: 

managers_exist= db.execute("SELECT 1 FROM sqlite_master WHERE 
      type='table' AND name= ?", "managers").length > 0 

More information here: http://stackoverflow.com/questions/32194515/sqlite-error-no-such-table